src/main/resources/default-cluster.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "num_workers": 1,
+    "cluster_name": "my-cluster",
+    "spark_version": "5.2.x-scala2.11",
+    "aws_attributes": {
+      "first_on_demand": 1,
+      "availability": "SPOT_WITH_FALLBACK",
+      "zone_id": "",
+      "instance_profile_arn": "",
+      "spot_bid_price_percent": 50,
+      "ebs_volume_type": "GENERAL_PURPOSE_SSD",
+      "ebs_volume_count": 1,
+      "ebs_volume_size": 100
+    },
+    "node_type_id": "m4.large",
+    "spark_env_vars": {
+      "PYSPARK_PYTHON": "/databricks/python3/bin/python3"
+    },
+    "autotermination_minutes": 10,
+    "artifact_paths": [],
+    "driver_node_type_id": "m4.large",
+    "spark_conf": {
+      "spark.databricks.delta.preview.enabled": "true",
```

Comment:
this is no longer required and should be removed

Reply:
Agree, let's do it within a separate commit